lib/Default/SmrPlayer.class.inc

```diff
@@ -217,8 +217,35 @@ class SmrPlayer extends AbstractSmrPlayer {
 		else {
 			throw new PlayerNotFoundException('Invalid accountID: '.$accountIDOrResultArray . ' OR gameID:'.$gameID);
 		}
+	}
+
+	// Get array of players who are sharing info with this player
+	// Only returns players who are in the same alliance
+	public function getSharingPlayers($forceUpdate=false) {
+		$results = array();
```

Comment:
If you use
```php
$results = array($this);
```
then code that uses this automatically has access to all of the players that they can get info about, rather than having to special case getting info about the current player

Reply:
I'm happy to do it this way, since all current uses of this function do indeed include the current player. Do you think the function name needs to change to reflect this?

Reply:
I think the name is ok